modules/nw-sriov-network-attachment.adoc

```diff
@@ -1,36 +1,34 @@
 // Module included in the following assemblies:
 //
-// * networking/multiple_networks/configuring-sr-iov.adoc
+// * networking/hardware_networks/configuring-sriov-net-attach.adoc
 
 // Because of an existing issue in go-yaml, the strings 'on' and 'off'
 // are interpreted as booleans, not strings. The SR-IOV admission controller
 // will reject 'spoofCheck' and 'trust' if the values are not strings.
 // So these values must be explicitly quoted in the YAML.
 // https://github.com/go-yaml/yaml/issues/214
 
-[id="configuring-sr-iov-networks_{context}"]
+[id="nw-sriov-network-attachment_{context}"]
 = Configuring SR-IOV additional network
 
-You can configure an additional network that uses SR-IOV hardware by creating a SriovNetwork Custom Resource (CR).
+You can configure an additional network that uses SR-IOV hardware by creating a SriovNetwork custom resource (CR).
 When you create a SriovNetwork CR, the SR-IOV Operator automatically creates a NetworkAttachmentDefinition CR.
 
 [NOTE]
 =====
-Do not modify or delete a SriovNetwork Custom Resource (CR) if it is attached to any Pods in the `running` state.
+Do not modify or delete a SriovNetwork CR if it is attached to any Pods in the `running` state.
 =====
 
 .Prerequisites
 
 * Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
-* Log in as a user with `cluster-admin` privileges.
+* An account with `cluster-admin` privileges.
```

Comment:
The only thing with this change now is that the first bullet is a full sentence, but this isn't. Either way, be consistent between the two.